jdi-light/src/main/java/com/epam/jdi/light/common/PropertyNames.java

```diff
@@ -0,0 +1,7 @@
+package com.epam.jdi.light.common;
+
+public class PropertyNames {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'PropertyNames' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4532609)